www/css/dark.css

```diff
@@ -31,6 +32,10 @@ body {
   color: var(--disabled-color) !important;
 }
 
+a.gray {
+  color: var(--disabled-color);
+}
```

Comment:
Why did you add this? This was not needed. There is another rule (`.gray`) that does this.

Reply:
merging commits. I had it you didn't so I just put it in